site/src/pages/WorkspacePage/WorkspaceReadyPage.tsx

```diff
@@ -124,6 +124,7 @@ export const WorkspaceReadyPage = ({
         workspace={workspace}
         handleStart={() => workspaceSend({ type: "START" })}
         handleStop={() => workspaceSend({ type: "STOP" })}
+        handleRestart={() => workspaceSend({ type: "START" })}
```

Comment:
A `restart` is essentially just a `start`, so I simplified here instead of building out a new state in our workspace machine.
There might be edge cases that require a `stop` before a `start`, but we do the exact same thing (start without stopping first) for `update` and we haven't had any complaints about that feature, so I figure we start simple.